package.json

```diff
@@ -40,7 +43,8 @@
     "@dhis2/cli-style": "^3.2.2",
     "enzyme": "^3.9.0",
     "enzyme-adapter-react-16": "^1.13.2",
-    "husky": "^2.3.0"
+    "husky": "^2.3.0",
+    "react-test-renderer": "^16.8.6"
```

Comment:
Personally I think I'd go with enzyme-to-json for your snapshot tests, since enzyme is already part of the codebase. Plus that way you can choose to do shallow rendering when necessary.